### ISSUE TITLE
`manual_let_else` fix recommendation is not valid Rust

### ISSUE THREAD

**Issue by leandrobbraga** [rust-lang/rust-clippy#16433]
### Summary

The `clippy::manual_let_else` lint is suggesting invalid Rust syntax. The suggested code is missing a comma in the struct destructuring pattern.

The suggestion is `let A { a: first_arg .. } = a else { return };` but the correct is `let A { a: first_arg, .. } = a else { return };`

https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=96cfffd8b24f6a5ab7ebd6c78bcb4bc

### Reproducer

Code:
```rust
#![deny(clippy::manual_let_else)]
struct A {
    a: u32,
    b: u32
}

fn main() {
    let a = A { a:1, b:1};
    let first_arg = match a {
        A { a, .. } => a,
        _ => return,
    };
}
```

Current output:
```
error: this could be rewritten as `let...else`
  --> src/main.rs:9:5
   |
 9 | /     let first_arg = match a {
10 | |         A { a, .. } => a,
11 | |         _ => return,
12 | |     };
   | |______^ help: consider writing: `let A { a: first_arg .. } = a else { return };`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/rust-1.92.0/index.html#manual_let_else
note: the lint level is defined here
```

Desired output:
```
error: this could be rewritten as `let...else`
error: this could be rewritten as `let...else`
  --> src/main.rs:9:5
   |
 9 | /     let first_arg = match a {
10 | |         A { a, .. } => a,
11 | |         _ => return,
12 | |     };
   | |______^ help: consider writing: `let A { a: first_arg, .. } = a else { return };`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/rust-1.92.0/index.html#manual_let_else
note: the lint level is defined here
```

### Version

```text
rustc 1.92.0 (ded5c06cf 2025-12-08)
binary: rustc
commit-hash: ded5c06cf21d2b93bffd5d884aa6e96934ee4234
commit-date: 2025-12-08
host: x86_64-unknown-linux-gnu
release: 1.92.0
LLVM version: 21.1.3
```

### Additional Labels

@rustbot label +l-suggestion-causes-error
